include/timer.h

```diff
@@ -44,16 +47,31 @@ static inline int64_t GetTicks(void)
 	        .count();
 }
 
+static inline int64_t GetTicksUs(void)
```

Comment:
We knock out voids in favour of `()`.. a bit less syntax baggage (and just as explicit given the absence of arguments mean the same thing).